previewers/betatest/X3DPreview.html

```diff
@@ -0,0 +1,37 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title class="textPreviewText">X3D Preview</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script type="text/javascript" src="js/xss.js"></script>
+    <script src="lib/jquery.i18n.js"></script>
+    <script src="lib/jquery.i18n.messagestore.js"></script>
+    <script src="lib/jquery.i18n.language.js"></script>
+    <script type="text/javascript" src="js/retriever.js"></script>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+        integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap-theme.min.css"
+        integrity="sha384-6pzBo3FDv/PJ8r2KRkGHifhEocL+1X2rVCTTkUfGk7/0pbek5mMa1upzvWbrUbOZ" crossorigin="anonymous">
+    <link type="text/css" rel="stylesheet" href="css/preview.css" />
+    <!-- X_ITE , latest via github -->
+    <script src="https://create3000.github.io/code/x_ite/latest/x_ite.min.js"></script>
```

Comment:
For production, they recommend using a cdn and picking a version. Right now the latest is at https://cdn.jsdelivr.net/npm/x_ite@9.5.2/dist/x_ite.min.js - does it make sense to switch this?

Reply:
We could also download it and put it in the lib directory, not sure what best practice is. But for now a cdn seems fine.

Reply:
Sounds good - I was more worried about /latest resulting in new code being loaded. I'm not sure we have a best practice overall - probably remote makes more sense for the version here, where people use it in hosted mode (getting the js from github.io versus a cdn isn't much difference), but people may want to use a local script if they're downloading everything here to run local.